src/components/auth/SignUpForm.jsx

```diff
@@ -1,18 +1,109 @@
-import React from 'react';
+import React, { useState } from 'react';
 import ReactRouterPropTypes from 'utils/react-router-prop-types';
 
-const SignUpForm = () => (
-  <div className="h-screen xl:h-auto flex py-5 xl:py-0 my-10 xl:my-0">
-    <div
-      className="my-auto mx-auto xl:ml-20 bg-white xl:bg-transparent px-5 sm:px-8 py-8 xl:p-0 rounded-md shadow-md xl:shadow-none w-full sm:w-3/4 lg:w-2/4 xl:w-auto"
-    >
-      <h2 className="intro-x font-bold text-2xl xl:text-3xl text-center xl:text-left">
-        Sign Up
-      </h2>
-      In Development...
-    </div>
-  </div>
-);
+  const SignUpForm = () => {
+
+    const [lastname, setLastname] = useState('');
+    const [firstname, setFirstname] = useState('');
+    const [surname, setSurname] = useState('');
+    const [company, setCompany] = useState('');
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [password2, setPassword2] = useState('');
```

Comment:
Dont use so many hooks if you dont need that. Rewrite this with
```
const [form, setForm] = useState({
  lastname: '',
  firstname: '',
  ...
})
```

Reply:
i have the error, uses this construction

Reply:
@Vital-jan most likely you are just not using it correctly